src/components/Form.js

```diff
@@ -1,21 +1,22 @@
 import { useRouter } from "next/router";
+import Input from "@/src/components/Input.js";
```

Comment:
You should use "Styled" as a prefix for Styled components that you import from your styles component folder. Additionally you have a Button component in your styles folder. Components do not belong in the styles folder. Only put styled components into the styles folder and follow the naming convention: "StyledButton".

Reply:
Will be resolved in later issue.